.github/workflows/docker.yml

```diff
@@ -0,0 +1,53 @@
+name: Docker
+
+on:
+  push:
+    branches:
+      - 'bahdotsh/test-workflow'
+  release:
+    types: [published]
+
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: name/persistencecore
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: checkout repo
+        uses: actions/checkout@v2
+
+      - name: build docker-image
+        if: github.event.push
+        run: make docker-build PROCESS=persistencecore DOCKER_IMAGE_NAME=persistenceone/persistencecore DOCKER_TAG_NAME=latest
```

Comment:
you can do
```
run: make docker-build
env:
  PROCESS: persistencecore
  DOCKER_IMAGE_NAME: persistenceone/persistencecore
  DOCKER_TAG_NAME: latest
```

Can build and push at the same time: `make docker-build-push` instead of build

Reply:
Ok